packages/manager/modules/pci/src/projects/project/data-processing/submit-job/submit-job.html

```diff
@@ -0,0 +1,82 @@
+        <oui-step-form
+            data-prevent-next="true"
+            data-valid="$ctrl.state.jobEngine.engine !== undefined"
+            data-header="{{ ::'data_processing_submit_job_stepper_job_type_title' | translate }}"
+        >
+            <pci-project-data-processing-submit-job-type-selector
+                data-job-engines="$ctrl.capabilities"
+                data-on-change-handler="$ctrl.onChangeJobTypeHandler"
+            >
+            </pci-project-data-processing-submit-job-type-selector>
+        </oui-step-form>
+        <!-- step 2 - select region -->
+        <oui-step-form
+            data-header="{{ ::'data_processing_submit_job_stepper_region_title' | translate }}"
+            data-prevent-next="true"
+            data-on-focus="$ctrl.updateAvailableRegions()"
+        >
+            <pci-project-data-processing-submit-job-region-selector
+                data-regions="$ctrl.regions"
+                data-display-selected-region="$ctrl.displaySelectedRegion"
+                data-on-change-handler="$ctrl.onChangeRegionHandler"
+            >
+            </pci-project-data-processing-submit-job-region-selector>
+        </oui-step-form>
+        <!-- step 3 - job sizing -->
+        <oui-step-form
+            data-header="{{ ::'data_processing_submit_job_stepper_sizing_title' | translate }}"
+            data-prevent-next="true"
+            data-on-submit="$ctrl.onSubmitJobSizingHandler(form)"
+        >
+            <pci-project-data-processing-submit-job-spark-sizing
+                data-templates="$ctrl.state.jobEngine.templates"
+                data-values="$ctrl.state.jobSizing"
+                data-validate="$ctrl.jobSizingValidate"
+            >
+            </pci-project-data-processing-submit-job-spark-sizing>
+        </oui-step-form>
+        <!-- step 4 - configure job -->
+        <oui-step-form
+            data-header="{{ ::'data_processing_submit_job_stepper_configure_title' | translate }}"
+            data-submit-text="{{ ::'data_processing_submit_job_button_label' | translate }}"
+            data-prevent-next="true"
+            data-on-submit="$ctrl.onSubmitJobHandler()"
+        >
+            <pci-project-data-processing-submit-job-spark-config
+                data-project-id="$ctrl.projectId"
+                data-region="$ctrl.state.region"
+                data-on-change-handler="$ctrl.onChangeJobConfigHandler"
+            >
+            </pci-project-data-processing-submit-job-spark-config>
+        </oui-step-form>
+    </oui-stepper>
+    <div class="submitjob__spinner" ng-if="$ctrl.isSubmitting">
+        <oui-spinner size="s"></oui-spinner>
+        <span class="spinner__label"
+            >{{ ::'data_processing_submit_job_spinner_label' | translate
+            }}</span
+        >
+    </div>
+    <div
+        class="submitjob__error"
+        ng-if="$ctrl.submitRetries>0 && !$ctrl.isSubmitting"
+    >
+        <span class="oui-icon oui-icon-error_circle"></span>
```

Comment:
`span` displaying an icon must have a `aria-hidden="true"` or else screen readers will try and read them
```suggestion
        <span class="oui-icon oui-icon-error_circle" aria-hidden="true"></span>
```